pkg/labels/integration_test.go

```diff
@@ -332,3 +338,66 @@ func TestRemoveLabelsTxnFailsIfLabelsChange(t *testing.T) {
 		t.Fatal("expected the transaction to fail when the labels are changed out from under it")
 	}
 }
+
+func TestMutateAndSelectHTTPApplicator(t *testing.T) {
```

Comment:
i should have made two commits for that maybe, sorry for making you jump through hoops